CMakeLists.txt

```diff
@@ -156,6 +155,8 @@ mark_as_advanced(BUILD_FOR_PACKAGING)
 cmake_dependent_option(FORCE_LEGACY_LIBBPF "Force usage of libbpf 0.0.9 instead of the latest version." False "ENABLE_PLUGIN_LIBBPF" False)
 mark_as_advanced(FORCE_LEGACY_LIBBPF)
 
+set(ENABLE_ACLK ${ENABLE_CLOUD})
```

Comment:
Why do we need this and simply not use `ENABLE_CLOUD` throughout?

Reply:
My intent here was to avoid changing C code in this PR (we use `ENABLE_ACLK` in essentially all places in the C code at the moment) because changing it would have made the PR massive and much harder to review, and thus theoretically would have delayed getting it merged. I plan to do a followup to convert the C code once this is merged, which will get rid of this bit of the CMakeLists file as well.